data/scripting/ui.lua

```diff
@@ -26,7 +34,10 @@ end
 --
 --    Make a nice moving transition to center on the 'map_pixel', which is a table
 --    that must contain 'x' and 'y' keys. The function will return as soon as
---    the transition is completed.
+--    the transition is completed. Usually this function is useful when
+--    scrolling back after using :meth:`scroll_to_field` or
+--    :meth:`wait_for_roadbuilding_and_scroll`. These functions are returning
+--    a table of 'map_pixel'.
```

Comment:
And what *is* a "table of map_pixel"? Explain please

Reply:
The first sentence states: "Make a nice moving transition to center on the 'map_pixel', which is a table that must contain 'x' and 'y' keys."

Reply:
Hm, then "a table of 'map_pixel'" would seem like a table whose keys are of unspecified type and whose values are tables of type `{x = …, y = …}`. At least the key type needs to be specified. E.g. if it's an array it could be phrased
~~~~
These functions return an :class:`array` of :class:`table`,
where each array entry represents a map pixel using the
same table structure as the ``map_pixel`` parameter.
~~~~

Reply:
Why are the keys unspecified? I think it's just one table like `{"x"=0, "y"=0}`. An array of tables makes no sense, imho.

What about:
~~~~
Usually this function is useful when  scrolling back after using
:meth:`scroll_to_field` or :meth:`wait_for_roadbuilding_and_scroll`.
~~~~

The argument to this function is explained in the first sentence, the return values form the other functions are are documented in the corresponding functions, e.g. [scroll_to_field](https://www.widelands.org/documentation/autogen_auxiliary_ui/#scroll_to_field)

I just wanted to link those functions because i remember to get confused about those 'map_pixels' and asked myself: "Everywhere are 'fields' used. I know how to get a 'field' but how do i get 'map_pixels'?"

Reply:
Changed it like so.

Reply:
Ah okay. This could be clarified a bit though to indicate that the argument is obtained from the return value:
```
Usually this function is useful when scrolling back after using
:meth:`scroll_to_field` or :meth:`wait_for_roadbuilding_and_scroll`.
The return value of these functions can be passed to
``scroll_to_map_pixel`` to move back to the previous location.
```
Perhaps even add an example (hope this is correct):
```
       local former_pixel = scroll_to_field(wl.Game().map:get_field(12, 34))
       sleep(5000)
       do_something()
       sleep(5000)
       scroll_to_field(former_pixel)
```

Reply:
Good idea :-)

Added lua code example.